samples/cpp/houghcircles.cpp

```diff
@@ -46,7 +46,8 @@ int main(int argc, char** argv)
 
     //![houghcircles]
     vector<Vec3f> circles;
-    HoughCircles(gray, circles, HOUGH_GRADIENT, 1, gray.rows/16,
+    HoughCircles(gray, circles, HOUGH_GRADIENT, 1,
+		         gray.rows/16, // change this value to detect circles with different distances each other
```

Comment:
small extra comments
- remove the extra whitespaces to have it positioned under gray --> better layout
-  should be `with different distances **to** each other`